testing/kuttl/e2e/pgbouncer/01-psql-connect.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: psql-connect
+spec:
+  template:
+    spec:
+      backoffLimit: 3
+      restartPolicy: "OnFailure"
+      containers:
+        - name: psql
+          securityContext:
+            allowPrivilegeEscalation: false
+            privileged: false
+            readOnlyRootFilesystem: true
+            runAsNonRoot: true
+          image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-14.1-0
+          command:
+            - "bash"
+            - "-c"
+            # Getting this chown/chmod correct was easier to do with copies of the certs
+            # and in this main container rather than an init
+            - "mkdir -p /pgdata/certs && cp /tmp/certs/* /pgdata/certs && chown postgres /pgdata/certs/* && chmod 600 /pgdata/certs/* && psql -c 'select version();'"
```

Comment:
🌱 The `install` command can do most of this in a single step: `-D` to create the target directories and `-m` to set the permissions. In this case, when globbing the source files, probably use `-t /pgdata/certs`.

`chown` is not usually necessary because the created files are owned by the container user.